<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: century gothic;
}
QFrame
{
	color: rgb(255, 255, 255);
	background-color: rgb(51, 51, 51);
	border-radius: 15px;
}
#Form {
	background-image: url(:/new/prefix1/loginBackground.jfif);
}
QLabel#label
{
	font: 700 24pt &quot;Century Gothic&quot;;
}
QLabel#label_2
{
	font: 8pt &quot;Century Gothic&quot;;
}
QLabel#label_3
{
	font: 700 9pt &quot;Century Gothic&quot;;
}
QLabel
{
	background: transparent;
}
QLineEdit
{
	background: transparent;
	border: none;
	font: 12pt &quot;Century Gothic&quot;;
	border-bottom: 1px solid #717072;
	color: rgb(255, 255, 255);
}
QPushButton
{
	color: rgb(255, 255, 255);
	background-color: rgb(170, 0, 255);
	border-radius: 10px;
}
QPushButton:Hover
{
	color: rgb(255, 255, 255);
	background-color: rgb(148, 0, 222);
}
QPushButton#forgotPasswordButton
{
	font: 700 8pt &quot;Century Gothic&quot;;
	border: 1px solid rgb(255, 0, 0);
	color: rgb(255, 255, 255);
	background-color: rgb(51, 51, 51);
}
QPushButton#forgotPasswordButton:Hover
{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 0, 0);
}
QCommandLinkButton
{
	color: rgb(255, 255, 255);
	font: 700 8pt &quot;Century Gothic&quot;;
}
QRadioButton
{
	color: rgb(255, 255, 255);
	font: 10pt &quot;Century Gothic&quot;;
}
QMessageBox
{
	background-color: rgb(51, 51, 51);
}
QMessageBox QLabel
{
	border: 0;
	border-radius: 6px;
	font-size: 10px;
	font-weight: bold;
	padding-left: 5px;
	padding-right: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
}
QMessageBox QPushButton
{
	font: 9pt &quot;Century Gothic&quot;;
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>30</y>
     <width>371</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="signupButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 700 24pt &quot;Century Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Create Account</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit_email">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_password">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_confPassword">
       <property name="styleSheet">
        <string notr="true">border: transparent;
border-bottom: 1px solid #717072;
font: 12pt &quot;Century Gothic&quot;;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Confirm Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;Century Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Should be 8-16 characters long</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="showPasswordButton">
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Century Gothic&quot;;</string>
       </property>
       <property name="text">
        <string>Show Password</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="backToLoginButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>430</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 10pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Back To Login</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
